create-envfile.py

```diff
@@ -44,9 +44,12 @@ def shuffle(chars):
 
 
 _simple_chars = shuffle(string.ascii_letters + string.digits)
-_strong_chars = shuffle(
-    string.ascii_letters + string.digits + string.punctuation.replace('"', "").replace("'", "").replace("`", "")
-)
+
+punctuation_without_dollar = string.punctuation.replace('$', '')
```

Comment:
Hi @amithkashyaps-ui 
In this way you are re-introducing the apos and quote as chars. i would suggest to do it as follows:
```
string.punctuation.replace('$', '').replace('"', "").replace("'", "").replace("`", "")
```
Please update it so we can merge it
Thanks :)